invenio/legacy/bibcheck/__init__.py

```diff
@@ -23,6 +23,13 @@
 This API lets other modules interact with bibcheck.
 """
 
+import warnings
+
+from invenio.utils.deprecation import RemovedInInvenio22Warning
+
+warnings.warn("Legacy BibCheck will be removed in 2.2.",
+              RemovedInInvenio22Warning)
+
```

Comment:
Note, there is an on going native checker implementation in: https://github.com/dset0x/invenio/tree/checker-port